bonsai/ethereum-relay/tests/solidity/contracts/ProxyTest.sol

```diff
@@ -19,44 +19,43 @@ contract Proxy {
     // Events
     event CallbackRequest(
         address account,
-        bytes32 image_id,
+        bytes32 imageId,
         bytes input,
-        address callback_contract,
-        bytes4 function_selector,
-        uint64 gas_limit
+        address callbackContract,
+        bytes4 functionSelector,
+        uint64 gasLimit
     );
 
     event ProofsSubmitted();
 
-    // Callback
+    /// @notice Data required to authorize a callback to be sent through the relay.
+    struct CallbackAuthorization {
+        bytes seal;
+        bytes32 postStateDigest;
+    }
```

Comment:
Instead of redefining the same structs I would recommend declaring `Proxy is IBonsaiRelay` and importing the structures from that file.

Reply:
After looking at this some more, this is part of a larger refactor that will need to happen, and not in this PR.